modules/dxp/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -1,70 +1,75 @@
 @import 'atlas-variables';
```

Comment:
Same no `atlas-variables`.

There is also a `_click-goal.scss` file that should be isolated.